lib/gettext-setup/version.rb

```diff
@@ -0,0 +1,3 @@
+module GettextSetup
+  VERSION = '0.24'
```

Comment:
I'm not sure that would work? As in, if I run `gem install gettext-setup`, then the git metadata won't be installed locally, right?